### ISSUE TITLE
Feature Request: Connect with Twitter

### ISSUE THREAD

**Issue by ttcath** [Automattic/wp-calypso#84341]
### What

I would like to share the post automatically to "X"(also know as Twitter) in case to share information to our followers in Twitter.

### Why

It was very useful in the past, but since last time, due to some reason, it no longer support.

**Comment by Aurorum** [comment]
> due to some reason, it no longer support.

This was dropped because Twitter disabled Jetpack's API access: https://jetpack.com/blog/the-end-of-twitter-auto-sharing/

I don't believe that it can be restored unless Twitter changes their decision.

**Comment by ttcath** [comment]
Yup. Also, i found that, if i am willing to post at X, at Twitter, it says: Something went wrong, but don’t fret — let’s give it another shot.  Looks like weird.
